### ISSUE TITLE
Create Favicons/Touch Icons

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#5]
Favicons/touch icons should be created to match recommendations (http://www.ucsb.edu/webguide/favicon-and-touch-icons)

**Comment by loganfranken** [comment]
URL of the page in the GitHub-hosted repo:
http://ucsb-wsg.github.io/favicon-and-touch-icons/

**Comment by loganfranken** [comment]
I think we need to complete #58 before we tackle this one (basically, update the standards regarding favicons/touch icons before creating the actual icons)

**Comment by tenken** [comment]
What would we want to include in a favicon/touch icon recommendation?

I typically use this on a site ...
https://www.drupal.org/project/responsive_favicons

But to better enlighten myself I did a quick google for "favicon 2015" and found this blog post:
https://realfavicongenerator.net/blog/favicon-why-youre-doing-it-wrong/

Which ultimately linked to this repository which has existed for 2 years and was updated 12 days ago. 
https://github.com/audreyr/favicon-cheat-sheet

Is it possible to take excerpts from this favicon cheatsheet and possibly link it into the webguide as a living-document resource ...

I'm not sure of the best approach here. Eg, the favicon cheatsheet seems to me to be:

1. comprehensive
2. kept up to date

If we take all this content and include it into the webgudie, we'd have to be moderating it constantly for updates (which I guess is doable by future contributions) ...
